src/parsers/manifestjson.js

```diff
@@ -524,8 +525,13 @@ export default class ManifestJSONParser extends JSONParser {
       typeof this.parsedJSON.applications?.gecko?.admin_install_only !==
       'undefined'
     ) {
-      this.collector.addError(messages.ADMIN_INSTALL_ONLY_PROP_RESERVED);
-      this.isValid = false;
+      if (!this.isEnterpriseAddon) {
+        this.collector.addError(messages.ADMIN_INSTALL_ONLY_PROP_RESERVED);
+        this.isValid = false;
+      } else if (!this.parsedJSON.applications?.gecko?.admin_install_only) {
```

Comment:
This block must be outdented by one level and combined with the flag.

Reply:
what do you mean?